chatarena/environments/conversation.py

```diff
@@ -29,13 +29,17 @@ def reset(self):
         self._next_player_idx = 0
         self.message_pool.reset()
 
-        init_timestep = TimeStep(observation=[],
-                                 reward=self.get_zero_rewards(),
-                                 terminal=False)
+        init_timestep = TimeStep(
```

Comment:
Ran black on this file (to fix my own code formatting) but changed the prev classes as well